test/spec_helper.rb

```diff
@@ -106,7 +106,7 @@ def load_resolvable(filename)
 require_relative "factories"
 
 # force loading all files to report proper code coverage
-Dir.chdir(libdir) { Dir["**/*.rb"].each { |f| require f } }
+Dir.chdir(libdir) { Dir["**/*.rb"].sort.each { |f| require f } }
```

Comment:
I think this forced loading is not needed anymore since we use the `SimpleCov.track_files` call above.